seismicpro/gather/utils/correction.py

```diff
@@ -1,50 +1,82 @@
 """Implements functions for various gather corrections"""
 
+import math
+
 import numpy as np
-from numba import njit
+from numba import njit, prange
+
+from .general_utils import mute_gather
 
 
 @njit(nogil=True, fastmath=True)
-def get_hodograph(gather_data, time, offsets, velocity, sample_rate, fill_value=0):
-    r"""Return gather amplitudes for a reflection traveltime curve starting from time `time` with velocity `velocity`,
-    assuming that it follows hyperbolic trajectory as a function of offset given by:
-    :math:`t(l) = \sqrt{t(0)^2 + \frac{l^2}{v^2}}`, where:
-        t(l) - travel time at offset `l`,
-        t(0) - travel time at zero offset,
-        l - seismic trace offset,
-        v - seismic wave velocity.
+def get_hodograph(gather_data, hodograph_times, sample_rate, interpolate=True, fill_value=np.nan, out=None):
+    """ Retrieve hodograph amplitudes from the `gather_data`.
```

Comment:
We ususally dont add space here
```suggestion
    """Retrieve hodograph amplitudes from the `gather_data`.
```